tfidf/preprocess.py

```diff
@@ -158,7 +183,7 @@ def normalize_term(self, text):
         text = self.handle_stopwords(text)
         return self.stem_term(text)
 
-    @lru_cache(maxsize=10000)
+    @cached(LRUCache(maxsize=10000))
```

Comment:
Also, and again - thought for future -, magic values like 10000 should be exposed to the client as config variables. Not a serious issue atm though.